index.html

```diff
@@ -183,35 +183,31 @@ <h3 id="unacceptablebehaviour">
+  <li><strong>Appreciate and Accommodate Our Similarities and Differences</strong>We come from many cultures and backgrounds, ways of life, and standard of behavior. Cultural differences can encompass everything from official religious observances to personal habits to clothing. Be respectful of people with different practices, attitudes, and beliefs. To help us achieve and maintain these high standards, each individual participant is expected to share responsibility for our work environment by adhering to the following behavioral guidelines:
+</li>
 	
-<p>To help us achieve and maintain these high standards, each individual participant is expected to share
-	responsibility for our work environment by adhering to the following behavioral guidelines:</p>
 	
-	<ol>
-		<li><strong>Respect.</strong> We are a large community of people who are passionate about our work, sometimes holding strong opinions beliefs. We are committed to dealing with each other with courtesy, respect, and dignity at all times. Misunderstandings and disagreements do happen. When conflicts arise, we are expected to resolve them maintaining that courtesy, respect, and dignity, even when emotions are heightened.</li>
+	<ul>
+		<li><strong>Respect.</strong> We are a large community of people who are passionate about our work, sometimes holding strong opinions and beliefs. We are committed to dealing with each other with courtesy, respect, and dignity at all times. Misunderstandings and disagreements do happen. When conflicts arise, we are expected to resolve them maintaining that courtesy, respect, and dignity, even when emotions are heightened.</li>
 		
-  <li>Never <a>harass</a> or <a>bully</a> anyone verbally, physically or 
-   <a data-lt="sexual harassment">sexually</a>.</li>
-  <li>Never <a>discriminate</a> on the basis of personal characteristics or 
-    group membership.</li>
-  <li>Communicate constructively and avoid <a>demeaning</a> or <a>insulting</a>
```

Comment:
This sentence is removed and the sense of it is partially duplicated elsewhere. In particular the introduction includes:

> W3C community participants should communicate constructively and avoid insulting, unwelcome or demeaning behavior.

In my view the introduction is a summary and every important thing in it should be duplicated by something that looks normative in the main body of the document, so looking through each of the three components of the removed statements in turn:

### Insulting behaviour or language

Avoidance of insulting behaviour is mainly in the unacceptable behaviour section, first bullet on "offensive comments", though that is a list that is broadly qualified, as opposed to this, which is entirely unqualified. Consider an edit to that unacceptable behaviour bullet to make _all_ offensive comments unacceptable, and list as "especially" examples the things after the "related to".

### Demeaning behaviour or language

There's no direct equivalent to this in the unacceptable behaviour section. The closest match is the bullet on microaggressions, and arguably the whole section could be seen as being about preventing demeaning behaviour, however neither of the two key aspects of demeaning, being loss of respect or dignity, are actually mentioned in the unacceptable behaviour section. Consider adding "avoid" statements about demeaning language to the unacceptable behaviour section.

### Communicate constructively

It is not clear what the motivation for removing this is, however in my view it is poorly defined and can be removed without loss. I'm not sure it is the place of this document to include it in any case. I think we should take the same line for this as for the removed text on "seek, accept and offer objective work criticism" on the following line.

Reply:
Re: Insulting Behaviour or Language, I feel this should remain qualified to avoid abuse of the CoC.

Similar for Demeaning Behaviour or Language the Microaggressions section is designed to not leave it open for interpretation.